### ISSUE TITLE
Gantt and WGAC 2.1 conformance

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#6423]
[Forum post](https://forum.bryntum.com/viewtopic.php?p=120313#p120313)

We are starting to see increased demands for compliance with accessibility standards from government customers. These demands spins out from the EU standard EN 301 549, which again builds upon W3C WCAG 2.1 recommendation.

Existing thoughts & measures
Now in relation to your UI components - in particular Gantt - have you had considerations and perhaps supportive measures to address specific recommendations from WCAG 2.1? If so, what are these?

While the recommendation covers a lot of areas, our main concern is that navigation, operation and all functionality should be executable/possible without the use of a mouse/pointing device. 

Short Cuts
From the documentation I see that custom keyboard mapping is supported, which is makes it possible for us to implement a mapping for the functionality in use. That is great  Perhaps you already have a recommended "standard" mapping?
Gantt/Customization/Keymap

Tab & Arrow Navigation
What about standard navigation inside the app? In the online examples, for example https://bryntum.com/products/gantt/examples/advanced/,  I find the tab and arrow key possibilities rather limited. I cannot tab into the chart itself - it seems to loop in the header section. I cannot use the arrows for 2D navigation either. Perhaps there is configuration that must enable such functionality?

**Comment by ExtAnimal** [comment]
Also, the request is that on initial `focusin`, the grid navigation moves to the first data cell instead of the first column header cell.
